paramgen_dest.go

```diff
@@ -1,25 +1,181 @@
+		"caCert": {
+			Default:     "",
+			Description: "caCert is the path to the CA certificate to use for TLS",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{},
+		},
+		"clientCert": {
+			Default:     "",
+			Description: "clientCert is the path to the client certificate to use for TLS",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{},
+		},
+		"clientKey": {
+			Default:     "",
+			Description: "clientKey is the path to the client key to use for TLS",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{},
+		},
+		"delivery.appID": {
+			Default:     "",
+			Description: "appID specifies the application that created the message.",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{},
+		},
+		"delivery.contentEncoding": {
+			Default:     "",
+			Description: "contentEncoding specifies the encoding of the message content.",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{},
+		},
+		"delivery.contentType": {
+			Default:     "text/plain",
+			Description: "contentType specifies the MIME type of the message content. Default is \"text/plain\".",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{},
+		},
+		"delivery.correlationID": {
+			Default:     "",
+			Description: "correlationID is used to correlate RPC responses with requests.",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{},
+		},
+		"delivery.deliveryMode": {
+			Default:     "2",
+			Description: "deliveryMode indicates the message delivery mode. Non-persistent (1) or persistent (2). Default is 2 (persistent).",
+			Type:        sdk.ParameterTypeInt,
+			Validations: []sdk.Validation{},
+		},
+		"delivery.expiration": {
+			Default:     "",
+			Description: "expiration specifies the message expiration time, if any.",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{},
+		},
+		"delivery.immediate": {
+			Default:     "false",
+			Description: "immediate indicates if the message should be treated as immediate. If true, the message is not queued if no consumers are on the matching queue.",
+			Type:        sdk.ParameterTypeBool,
+			Validations: []sdk.Validation{},
+		},
+		"delivery.mandatory": {
+			Default:     "false",
+			Description: "mandatory indicates if the message is mandatory. If true, tells the server to return the message if it cannot be routed to a queue.",
+			Type:        sdk.ParameterTypeBool,
+			Validations: []sdk.Validation{},
+		},
+		"delivery.messageTypeName": {
+			Default:     "",
+			Description: "messageTypeName specifies the message type name.",
+			Type:        sdk.ParameterTypeString,
+			Validations: []sdk.Validation{},
+		},
+		"delivery.priority": {
+			Default:     "0",
+			Description: "priority specifies the message priority. Ranges from 0 to 9. Default is 0.",
```

Comment:
we can also add validation for this param, greater than -1 and less than 10, check [paramgen docs](https://github.com/ConduitIO/conduit-commons/tree/main/paramgen#:~:text=lt%20or%20less,less%2Dthan%3D200.2%22%60)